legacy_ibc_testing/core/events.go

```diff
@@ -0,0 +1,48 @@
+package core
+
+import (
+	"strconv"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+/*
+TODO: Remove after upgrading to ibc-go v5
+legacy_ibc_testing is temporarily copied into the interchain-security repository for the purpose of testing only.
+The e2e test suites rely on modifications to ibc-go's test framework that cannot be back-ported to the canonical version that ics will rely on.
+These files will be deprecated once ICS is able to upgrade to ibc-go v5.
```

Comment:
Maybe we can add links to the IBC fork for context? If you do so, please add it to every instance of this docstring.

Alternatively, we could include a Readme.md in the folder to keep track of all the nuances.

Reply:
I think a Readme would be more comprehensive, happy to add it in this PR or in a subsequent pr